app/suwon-scrape/page.tsx

```
@@ -0,0 +1,118 @@
+          <input
+            type="password"
+            value={password}
+            onChange={e => setPassword(e.target.value)}
+            style={{ marginLeft: '10px' }}
+          />
+        </label>
+      </div>
+      <button onClick={startScrape} style={{ marginRight: '10px' }}>
+        크롤링 시작
+      </button>
+      <div style={{ marginTop: '20px' }}>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)